evalml/data_checks/null_data_check.py

```diff
@@ -180,6 +243,54 @@ def validate(self, X, y=None):
                 ).to_dict()
             )
 
+        if below_highly_null_cols:
+            results["warnings"].append(
+                DataCheckWarning(
+                    message="Columns {} have null values".format(
+                        (", ").join(
+                            ["'{}'".format(str(col)) for col in below_highly_null_cols]
+                        )
+                    ),
+                    data_check_name=self.name,
+                    message_code=DataCheckMessageCode.COLS_WITH_NULL,
+                    details={
+                        "columns": list(below_highly_null_cols),
+                    },
+                ).to_dict()
+            )
+
+            impute_strategies_dict = {}
+            for col in below_highly_null_cols:
+                col_in_df = X.ww[col]
+                impute_strategies_dict[col] = {
+                    "impute_strategy": {
+                        "categories": ["mean", "mode"]
+                        if col_in_df.ww.schema.is_numeric
+                        else ["mode"],
+                        "type": "category",
+                        "default_value": "mean"
+                        if col_in_df.ww.schema.is_numeric
+                        else "mode",
```

Comment:
Our linter definitely made this formatting, but it's kind of hard to read. Does it look cleaner/clearer if you pull the if/else logic out of this block?

Reply:
Yes, I think so! 😁 
![image](https://user-images.githubusercontent.com/5422235/149368459-cae16fc4-12d1-4eab-ad42-080f6cc67cc2.png)